server/src/repositories/student.repository.ts

```diff
@@ -85,6 +88,7 @@ export class StudentRepository extends AbstractRepository<Student> {
       id: entity.id,
       githubId: entity.githubId,
       name: userService.createName(entity),
+      mentorGithub: entity.mentorGithub,
```

Comment:
please rename here and in other places to `mentorGithubId` for consistency (we used in some places already)

Reply:
done